python/tk_desktop/desktop_window.py

```diff
@@ -818,10 +848,16 @@ def report_progress(percentage, message):
             # comes from.
```

Comment:
I think my comment induced you in error here. My comment should have read

`If no pipeline configuration is in the user settings, we will let the bootstrap pick the right pipeline configuration for the first launch.`

See next comment.